pkgs/servers/misc/virtiofsd/default.nix

```diff
@@ -21,6 +21,15 @@ rustPlatform.buildRustPackage rec {
 
   buildInputs = [ libcap_ng libseccomp ];
 
+  postBuild = ''
```

Comment:
doesn't really matter, postConfigure would make more sense, as this is something that would be handled in the configure phase of a build system.